ci/requirements-conversion.txt

```diff
@@ -1,53 +1,53 @@
 # use update-requirements.py to update this file
 
-absl-py==0.9.0            # via tensorboard, tensorflow
+absl-py==0.10.0           # via tensorboard, tensorflow
 astunparse==1.6.3         # via tensorflow
 cachetools==4.1.1         # via google-auth
 certifi==2020.6.20        # via requests
 chardet==3.0.4            # via requests
 decorator==4.4.2          # via networkx
-defusedxml==0.6.0         # via -r ${INTEL_OPENVINO_DIR}/deployment_tools/model_optimizer/requirements.txt
-future==0.18.2            # via -r tools/downloader/requirements-caffe2.in
+defusedxml==0.6.0         # via -r ${INTEL_OPENVINO_DIR}/deployment_tools/model_optimizer/requirements_caffe.txt, -r ${INTEL_OPENVINO_DIR}/deployment_tools/model_optimizer/requirements_mxnet.txt, -r ${INTEL_OPENVINO_DIR}/deployment_tools/model_optimizer/requirements_onnx.txt, -r ${INTEL_OPENVINO_DIR}/deployment_tools/model_optimizer/requirements_tf2.txt
```

Comment:
do we really need deps for tf2, while all our models if tf1 version?
does it no break anything?

Reply:
That's why we need them, to ensure it doesn't break anything. 🙂

TF2 is supported by MO, and with Python 3.8+ it's the only supported version, so we need to make sure our old models still work.